Formula/y/yices2.rb

```diff
@@ -0,0 +1,36 @@
+class Yices2 < Formula
+  desc "Yices SMT Solver"
+  homepage "https://yices.csl.sri.com/"
+  url "https://github.com/SRI-CSL/yices2/archive/refs/tags/Yices-2.6.5.tar.gz"
+  sha256 "46a93225c1e14ee105e573bb5aae69c8d75b5c65d71e4491fac98203cb0182f3"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "gmp"
+  depends_on "gperf"
```

Comment:
should be build dep?
```suggestion
  depends_on "gperf" => :build
```